### PR TITLE
6 product page

### PR DESCRIPTION
**Pull request by dousdogukan**
### Description

implement **Products page** 
implement **CMethod** and **CIteamCard**

### Checklist

- [ ] Tested changes locally.
- [ ] Ensured code follows project standards.
- [ ] Updated documentation.
- [ ] Added necessary tests.

### Additional Notes

The product modal will be created in the subsequent PRs.
The coin labels will be added after merging the previous PR.
In CItemCard, there are two states: product and transaction.
After merging this PR, the transaction styles will be added to CItemCard.

for #6

### CODE REVIEW

File: src/app/products/page.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import CPageCard from '@/components/CPageCard';
+import { products } from '@/constants/productsList';
+
+const Products = () => {
+  const pageTitle = (
+    <div className="flex justify-between items-center w-full">
+      <h1>Product</h1>
+      <CButton variant="add" text="Add product" className="!w-[145px] text-base" />
+    </div>
+  );
```
Comment (ceydae) [inline src/app/products/page.tsx:12]: Move this out of the Proucts component

File: src/assets/Stream.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const Icon = () => (
+  <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M2.66797 4.66699H10.668" stroke="#98A2B3" stroke-width="1.5" stroke-linecap="round" />
+    <path d="M4.66797 8H14.0013" stroke="#98A2B3" stroke-width="1.5" stroke-linecap="round" />
+    <path d="M2.66797 11.333H10.668" stroke="#98A2B3" stroke-width="1.5" stroke-linecap="round" />
+  </svg>
```
Comment (ceydae) [inline src/assets/Stream.tsx:6]: invalid props, use camelCase

**Review by ceydae — CHANGES_REQUESTED** [review]
`Module not found: Can't resolve 'clsx'`

The app does not run

File: src/components/CItemCard/index.tsx
Comment (ceydae) [inline src/components/CItemCard/index.tsx:1]: This is a wrong way of making a component for both Product and Transaction.

**Review by ceydae — CHANGES_REQUESTED** [review]
remove space between # and id

**Review by ruveyda-kisla — APPROVED** [review]

File: src/components/CItemCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+import CMethod, { MethodType } from '../CMethod';
+
+export type ItemVariantType = 'product' | 'transaction';
+
+export interface CItemCardProps {
+  title: string;
+  image: string | StaticImageData;
+  className?: string;
+  onClick?: () => void;
+  method: MethodType;
```
Comment (ceydae) [inline src/components/CItemCard/index.tsx]: remove `method` type from props of this component

**Review by ceydae — CHANGES_REQUESTED** [review]

**Pull request merged**